_dev/apps/ui/src/views/onboarding-page.vue

```diff
@@ -2,39 +2,117 @@
   <div class="pt-2 container">
     <div class="row">
       <AlertCmp />
+      <MonetizationBannerInformation v-if="!googleAccountIsOnboarded && !merchantIsSuscribed" />
+      <b-alert
+        v-if="displayBannerSuccessMonetization"
+        show
+        variant="success"
+        class="border border-success mx-3 w-100"
+        dismissible
+      >
+        {{ $t('banner.monetization.bannerSuccessBilling') }}
+      </b-alert>
     </div>
+
+    <b-button
+      variant="primary"
+      @click="toogleSubscription"
+    >
+      {{ !merchantIsSuscribed ? "Is subribed" : "Not subscribed" }}
```

Comment:
Missing translation

Reply:
This is a test button, which will be removed before it goes live.